docs/genai/deployment/genai_serving.md

```diff
@@ -0,0 +1,190 @@
+(genai-serving)=
+# Serving gen AI models
+
+Serving a gen AI model is, in essence, the same as serving any other model. The main differences are with the inputs and outputs, which are usually unstructured (text or images), and the model is usually a transformer model. With MLRun you can serve any model, including pretrained models from the [Hugging Face model hub](https://huggingface.co/docs/hub/en/models-the-hub) as well as models that are fine-tuned with MLRun.
+
+Another common use case is to serve the model as part of an inference pipeline, where the model is used as part of a larger pipeline that includes data preprocessing, model execution, and post-processing. This is covered in the {ref}`gen AI serving graph section <genai-serving-graph>`.
+
+
+## Serving using the function hub
+
+The function hub has a serving class called [`hugging_face_serving`](https://www.mlrun.org/hub/functions/master/hugging_face_serving/) to run Hugging Face models. The following code shows how to import the function to your project:
+
+```python
+hugging_face_serving = project.set_function("hub://hugging_face_serving")
+```
+
+Next, you can add a model to the function using this code:
+
+```python
+
+hugging_face_serving.add_model(
+    'mymodel',
+    class_name='HuggingFaceModelServer',
+    model_path='123',  # This is not used, just for enabling the process.
```

Comment:
I'm not sure what this comment even means. If it's not used, is it mandatory?

Reply:
Unfortunately yes, I opened a ticket on that